### PR TITLE
Add nidcpower API defaults and enums

### LINKED ISSUE #300: Add default parameter metadata to NI-DCPower
Go through the API with a domain expert, and decide for each function which parameters should have default values in the Python API and what those values should be.

This needs to be done carefully, because in Python, all parameters with default values must go at the end of the parameter list. So changing this later would likely be a source breaker for client code.

This will most likely be a subset of Recommended/Optional input terminals on corresponding LabVIEW VIs.

### PR DESCRIPTION
**Pull request by Fladolcetta**
[x ] This contribution adheres to [CONTRIBUTING.md]

### What does this Pull Request accomplish?

Adds defaults and new required enums for some functions

### List issues fixed by this Pull Request below, if any.

* Fixes #300 
* Brings API closer to our desired final API

### What testing has been done?

Built and ran unit tests

### CODE REVIEW

File: docs/nidcpower/functions.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ nidcpower.Session methods
 
         
 
-    :type aperture_time: float
+    :type aperture_time: :py:data:`nidcpower.ApertureTimeUnits`
```
Comment (marcoskirsch) [inline docs/nidcpower/functions.rst]: This used to be float?
Hopefully that was a (now fixed) bug.

File: generated/nidcpower/session.py [hunk 2]
```diff
@@ -2536,11 +2536,13 @@ def configure_aperture_time(self, aperture_time, units):
...
         '''
-        error_code = self._library.niDCPower_ConfigureApertureTime(self._vi, self._repeated_capability.encode(self._encoding), aperture_time, units)
+        if type(aperture_time) is not enums.ApertureTimeUnits:
+            raise TypeError('Parameter mode must be of type ' + str(enums.ApertureTimeUnits))
+        error_code = self._library.niDCPower_ConfigureApertureTime(self._vi, self._repeated_capability.encode(self._encoding), aperture_time.value, units)
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def fetch_multiple(self, timeout, count):
+    def fetch_multiple(self, count, timeout=1):
```
Comment (marcoskirsch) [inline generated/nidcpower/session.py]: If the parameter is float, you should specify the literal as 1.0 so the runtime doesn't need to convert.
Applies everywhere.

File: generated/nidcpower/session.py [hunk 3]
```diff
@@ -2487,7 +2487,7 @@ def get_error_description(self, error_code):
 
     ''' These are code-generated '''
 
-    def configure_aperture_time(self, aperture_time, units):
+    def configure_aperture_time(self, units, aperture_time=enums.ApertureTimeUnits.SECONDS):
```
Comment (marcoskirsch) [inline generated/nidcpower/session.py]: This is wrong.

File: generated/nidcpower/session.py [hunk 4]
```diff
@@ -3407,7 +3409,7 @@ def _set_attribute_vi_string(self, attribute_id, attribute_value):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def set_sequence(self, values, source_delays, size):
+    def set_sequence(self, source_delays, size, values=''):
```
Comment (marcoskirsch) [inline generated/nidcpower/session.py]: values should be None or [], not blank string. values is an array!

File: generated/nidcpower/session.py [hunk 5]
```diff
@@ -3556,7 +3558,7 @@ def commit(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def configure_digital_edge_measure_trigger(self, input_terminal, edge):
+    def configure_digital_edge_measure_trigger(self, edge, input_terminal=enums.DigitalEdge.RISING):
```
Comment (marcoskirsch) [inline generated/nidcpower/session.py]: You have these off by one errors in many places. The edge is rising, not the input terminal!

File: generated/nidcpower/session.py (hunk 2 above)
Comment (Fladolcetta) [reply generated/nidcpower/session.py]: Fixed

File: generated/nidcpower/session.py (hunk 4 above)
Comment (Fladolcetta) [reply generated/nidcpower/session.py]: Fixed

File: generated/nidcpower/session.py (hunk 3 above)
Comment (Fladolcetta) [reply generated/nidcpower/session.py]: Fixed

File: generated/nidcpower/session.py (hunk 5 above)
Comment (Fladolcetta) [reply generated/nidcpower/session.py]: Fixed

File: generated/nidcpower/session.py [hunk 6]
```diff
@@ -3407,7 +3409,7 @@ def _set_attribute_vi_string(self, attribute_id, attribute_value):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def set_sequence(self, values, source_delays, size):
+    def set_sequence(self, source_delays, size, values=None):
```
Comment (marcoskirsch) [inline generated/nidcpower/session.py]: size should not be here

File: src/nidcpower/metadata/functions_addon.py [hunk 7]
```diff
@@ -70,6 +78,8 @@
     'revision_query':               { 'parameters': { 1: { 'size': {'mechanism':'fixed', 'value':256}, }, 
                                                       2: { 'size': {'mechanism':'fixed', 'value':256}, }, }, }, # From documentation
     'GetChannelName':               { 'parameters': { 3: { 'size': {'mechanism':'ivi-dance', 'value':'bufferSize'}, }, }, },
+    'SetSequence':                  { 'parameters': { 1: { 'size': {'mechanism':'passed-in', 'value':'Size'}, }, }, },
```
Comment (marcoskirsch) [inline src/nidcpower/metadata/functions_addon.py]: The problem above is probably that you misspelled this parameter.
The code-generator should die if mechanism is "passed-in" _and_ value is not found in the parameter list. If you confirm this was the issue, we can look into adding that check.

Comment (Fladolcetta) [reply src/nidcpower/metadata/functions_addon.py]: This was correctly spelled, but will be resolved by the len feature @texasaggie97 is working on.

File: generated/nidcpower/session.py (hunk 6 above)
Comment (Fladolcetta) [reply generated/nidcpower/session.py]: This will be resolved by the len feature @texasaggie97 is working on.

**Pull request merged**